src/app/base/components/NodeActionMenu/NodeActionMenu.test.tsx

```diff
@@ -113,20 +119,35 @@ describe("NodeActionMenu", () => {
     await openMenu();
 
     expect(getActionButton(NodeActions.DEPLOY)).toBeInTheDocument();
-    expect(queryActionButton(NodeActions.DEPLOY)).not.toBeDisabled();
+    expect(queryActionButton(NodeActions.DEPLOY)).not.toHaveAttribute(
+      "aria-disabled",
+      "true"
+    );
     expect(getActionButton(NodeActions.RELEASE)).toBeInTheDocument();
-    expect(getActionButton(NodeActions.RELEASE)).toBeDisabled();
+    expect(getActionButton(NodeActions.RELEASE)).toHaveAttribute(
+      "aria-disabled",
+      "true"
+    );
```

Comment:
There's a few more examples of `toBeAriaDisabled(` not being used instead of `toHaveAttribute(`